src/collectd.conf.in

```diff
@@ -1902,3 +1902,23 @@
 #    </Type>
 #  </Host>
 #</Plugin>
+
+#@BUILD_PLUGIN_WMI_TRUE@LoadPlugin "wmi"
+#<Plugin wmi>
+#  <Query>
+#    Statement "select * from Win32_PerfFormattedData_PerfOS_Processor where Name = '_Total'"
+#    InstancePrefix "cpu"
+#    InstancesFrom "Name"
```

Comment:
Nit: In this example query, the Name field is static. Could you think of an example where this is not the case? E.g. Name != '_Total' (assuming that this would provide per-CPU metrics)?

Reply:
Yep, Name != '_Total' would work here. Updated.